src/design-system/styles/template/block-areas/index.njk

```diff
@@ -0,0 +1,131 @@
+---
+title: Block areas – Template
+
+stylesheets:
+- ./block-areas-annotate.css
+---
+
+{% extends "template.njk" %}
+
+{% set bodyClasses = "app-example-page" %}
+
+{% block head -%}
+  <meta name="robots" content="noindex, nofollow">
+
+  <!--[if !IE 8]><!-->
+    <link href="/{{ fingerprint['design-system/stylesheets/main.css'] }}" rel="stylesheet" media="all" />
+  <!--<![endif]-->
+  <!--[if lt IE 9]>
+    <link href="/{{ fingerprint['design-system/stylesheets/main-ie8.css'] }}" rel="stylesheet" media="all" />
+    <script src="/{{ fingerprint['design-system/javascripts/ie.js'] }}"></script>
+  <![endif]-->
+  <script src="/design-system/javascripts/vendor/modernizr.js"></script>
+
+  {% for stylesheet in stylesheets %}
+    <link href="{{ stylesheet }}" rel="stylesheet" media="all" />
+  {%- endfor %}
+{%- endblock %}
+
+{% block bodyStart -%}
+  {# Since we can't show the blocks in the `<head>`, we pretend by creating a section in `bodyStart` #}
+  <div data-block="headIcons">
```

Comment:
I'm not entirely sure about using `data` elements here. Do we know if all assistive tech read out the `data-block` here? It provides important info as to what the content is. It might be better to set the relevant text in a div and style it appropriately.

Reply:
I have changed these to be regular content and classes.